source/reference_guide/concepts_info/concepts/hq_attack.rst

```diff
@@ -0,0 +1,15 @@
+.. _concept_hq_attack:
+
+HQ attack system
+============================================
+
+.. card::
+   :class-card: sd-mt-3
+   :class-header: header-2-light
+
+   Description
+   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+   HQ attacks can now only be launched once an enemy faction has full knowledge of your HQ. Knowledge is currently added when a traitor mission is failed, or when a support is called against a target near HQ. Knowledge diminishes slowly over time as long as it hasn't reached 1. Moving HQ will reset knowledge, as long as their wasn't a previous HQ nearby recently.
+
+   The attacks themselves are still specops air, but benefit from improvements to combat behaviour. Support artillery and UAVs are also used to assist.
```

Comment:
Changed, HQ attacks can now include ground components, although less than a major attack.

Reply:
i tried to put that into the sentence please check after i commited the changes (will ping you)